<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_create">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать вектор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pushBack">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>push_back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_popBack">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>pop_back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resize">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>resize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_at">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>at</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_size">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_value">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_newSize">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_index">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>151</y>
      <width>501</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_size">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_capacity">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>размер вектора</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>добавляемый элемент</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>размер при resize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>доступ к элементу</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
